### DISCUSSION TITLE
Log files?

### DISCUSSION THREAD

**Discussion by Jinxzy13** [TheLastGimbus/GooglePhotosTakeoutHelper#165] [I NEED HELP] [answered]
### **Seems the program work fine once I extracted myself and used --fix on the folder**

Where is the log files for this?

I downloaded and started on Ubuntu 20.04 and selected all the options to start.

The file is 11gb and after 20 minutes extracting I just get a message saying 'Killed' 

I tried twice on the same file.

I have 100gb of google data I am wanting to move eventually but figured I would try on 2022 to start
![Screenshot from 2023-01-30 16-22-53](https://user-images.githubusercontent.com/123977173/215541154-28e31314-ebe9-4775-b1e9-724fdaf40822.png)

**Comment by TheLastGimbus** [comment, marked as answer]
Hi! There are no log files, only info is in the console

> Worked with --fix

Huh, very interesting. That's because --fix works by iterating over all files as-is, whereas standard mode creates the list of Media data classes that holds info about everything. Turns out that it may be crashing where there are too much: #164

That sucks. Will need to fix that!

Best solution for you would be the same as for @matt-boris : https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/issues/164#issuecomment-1407386893

**Reply by Jinxzy13** [reply]
Thank you, the takeout feature is so flawed. None of the numbers match with what’s extracted to what’s selectable. Ive no idea how to know I’ve got everything. 

That’s out of scope of your tool though I know ha thank you for the reply.

**Comment by TheLastGimbus** [comment]
# Fixed in https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/releases/tag/v3.3.3 :tada:
